core/unix/module_elf.h

```diff
@@ -42,6 +42,11 @@
 #    define DT_RELR 36
 #endif
 
+/* XXX: Workaround for EM_RISCV not being defined in elf.h on RHEL-7. */
```

Comment:
XXX means there is still work to do (though less work or less urgently than TODO) so it should just be removed here.

Reply:
Fixed.